### DISCUSSION TITLE
[A-L] (2023/24) Foglio 12 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#337] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-28 132221](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/e46697c4-47ff-40bf-8fe2-4d82366a3836)

**Comment by Elia-Belli** [comment]
$W$ è un sottospazio se:
- La somma di due vettori $x,x'\in W$ si trova in $W$: 
  sappiamo che $x_1+x_2+x_3=0$ e $x_1'+x_2'+x_3'=0$, la somma di $x+x'$ è il vettore $(x_1+x_1',x_2+x_2',x_3+x_3')$,
  osserviamo che per commutatività $x_1+x_1'+x_2+x_2'+x_3+x_3'=x_1+x_2+x_3+x_1'+x_2'+x_3'=0+0=0\Rightarrow x+x'\in W$
- Il prodotto di un vettore $w\in W$ per uno scalare si trova in $W$:
  quindi $\lambda\cdot x=(\lambda x_1,\lambda x_2,\lambda x_3)$, osserviamo che per distributività $\lambda x_1+\lambda x_2+\lambda x_3=\lambda (x_1+x_2+x_3)=\lambda \cdot 0=0$, allora $\lambda x\in W$.
Abbiamo verificato che $W\leq \mathbb{R}^3\ \square$

$dim\ W=2$ e una sua base è:

$$\mathcal{B}(W)=\Bigg( \begin{pmatrix}1\\
-1\\
0\end{pmatrix},\begin{pmatrix}1\\
0\\
-1\end{pmatrix}\Bigg)$$

**Comment by CiottoloMaggico** [comment]
Analogo @Elia-Belli con ragionamenti punto due
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/fc834858-50a6-4115-96c4-515bdfe3ce1d)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/4cf85018-81c8-43dc-8afb-b8dad3aaee91)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/cca051ae-59a5-4ebf-bf1c-3aedd2ad426d)
